101-backup-la-reporting/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Specify the workspace name"
+      }
+    },
+    "workspaceLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Specify the workspace region"
+      },
+      "defaultValue": "West Europe"
```

Comment:
remove this and use the parameters file (you already are)

Reply:
Updated